translations/README.fr.md

```diff
@@ -0,0 +1,73 @@
+# Générateur de Résumé GitHub
+
+![Résumé GitHub pour React](../src/assets/readme/screenshot.png)
+
+## Table des Matières
+
+- [A propos](#about)
+- [Compilé avec](#built-with)
+- [Installation](#installation)
+- [Demo](#live-demo)
+
+<br>
+
+### 🌎 *Lire dans [d'autres langages](./Translations.md)*
+
+
+<h2 id='about'>🤔 A propos</h2>
+
+L'objectif de ce repository est de générer des "Résumé GitHub" pour chaque utilisateur.
+
+<h2 id='built-with'>🛠️ Compilation avec</h2>
+
+- [React JS](https://reactjs.org/) - Front-End JavaScript library
+- [Material UI](https://material-ui.com/) - React UI Framework
+
+<h2 id='installation'>Installation</h2>
+
+Pour configurer l'application pour le développement sur votre ordinateur local, veuillez suivre les inscrutions ci-dessous :
+
+1. Cloner ce repository sur votre machine
+
+```bash
+git clone https://github.com/sabesansathananthan/React-GitHub-Resume.git
+cd React-GitHub-Resume
+```
+
+2. Installer les packages
+
+    Si vous utilisez `npm`
+    ```bash
+    npm install
+    ```
+
+    or
+
+    Si vous utilisez `yarn`
+    ```bash
+    yarn
+    ```
+
+3. Execution sur un serveur en développement
+
+    Si vous utilisez `npm`
+    ```bash
+    npm start
+    ```
+
+    or
+
+    Si vous utilisez `yarn`
+    ```bash
+    yarn start
+    ```
+
+4. Visiter <http://localhost:3000>
+
+<h2 id='live-demo'><a href="https://react-github-resume.vercel.app/">Demo Direct</a></h2>
+
+[![Deployer avec Vercel](https://vercel.com/button)](https://vercel.com/new/git/external?repository-url=https://github.com/sabesansathananthan/React-GitHub-Resume)
+
+## 📄 License
+
+Ce projet est sous licence MIT - voir le fichier [LICENSE](../LICENSE) pour plus de détails.
```

Comment:
Change [LICENSE] as well

Reply:
Noted

Reply:
If other things are missing, do not hesitate, I put star